### DISCUSSION TITLE
Spotify customization

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1334] [Q&A] [answered]
I installed spotify and spicetify. And ran the script wallbashspotify.sh but I got the error 
"error Theme "Sleek" not found",I got the same error when i ran spicetify apply but there is a folder Sleek in /home/.config/spicetify/Themes/Sleek. Now spotify is opening but there it shows a  blank screen.

**Comment by kRHYME7** [comment]
what's the output if you run ``` wallbashspotify.sh --reset ``` ?

**Reply by sanjyay** [reply]
I got the same error
![240418_19h42m30s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/102979855/2dbbc366-a408-4d73-8c40-4322de7cc77f)

**Reply by kRHYME7** [reply]
? what's the deal of running it as sudo ? 😭

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/b499c289-3f78-4ee8-9dc0-87893f68f181)

**Reply by sanjyay** [reply]
because of this permission denied command I thought sudo will do the trick
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/d044f8a7-30c2-41e3-9f74-faeeecf6c058)
/opt/spotify has no folder named app but only a folder named Apps which I am not able to enter 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/56667ab6-5223-454e-90f0-d51ce975e3cf)
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/49c6fc00-47c9-48b9-a8b2-2035e5378072)

**Reply by kRHYME7** [reply]
You can do this manually, it is the polkit that is failing in your case (maybe needs a reboot) 

This is the polkit  that should be asking for permissions.
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f702819d-7643-4130-94c0-fdb46c5c0b90)


So do this manually instead. This is explicitly mentioned on the spicetify IG to give some permissions to these files/dir
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/4cebea5a-7678-4241-87d3-9dc48a466d73)

```
       sudo chmod a+wr /opt/spotify
        sudo chmod a+wr /opt/spotify/Apps -R
```

**Reply by sanjyay** [reply]
Waybar wasn't loading so i ran restore_cfg.sh script and did what you had mentioned but got these errors
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/80f60ace-e15b-41a4-8b01-668073b4b173)

I also tried rebooting still the same

**Reply by kRHYME7** [reply]
can you ls -al on that directory ? 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/32330010-55b7-4a61-8bf0-bc6c0a8c57b1)

Also I do not have that file too. 



https://spicetify.app/docs/advanced-usage/installation/
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/f295ae1c-f815-49ec-9012-4df808243923)

This exact lines did not work?  (be sure to kill spotify in the background too)
```
       sudo chmod a+wr /opt/spotify
        sudo chmod a+wr /opt/spotify/Apps -R
```

**Reply by sanjyay** [reply, marked as answer]
I resolved the problem . I deleted /opt/spotify and then did a fresh install of both spotify and spicetify-cli and followed the steps @kRHYME7 suggested. Thanks for such a quick reply and continued assistance :)
